src/core/qgsvectorlayer.h

```diff
@@ -1568,7 +1569,7 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
      * changes can be discarded by calling rollBack().
      * \since QGIS 3.16
      */
-    Q_INVOKABLE QgsGeometry::OperationResult splitFeatures( const QgsCurve *curve, bool preserveCircular = false, bool topologicalEditing = false );
+    Q_INVOKABLE QgsGeometry::OperationResult splitFeatures( const QgsCurve *curve, QgsPointSequence &topologyTestPoints SIP_OUT, bool preserveCircular = false, bool topologicalEditing = false );
```

Comment:
this is an API break as the result in Python is becoming a tuple

Reply:
Is it an API break if it is not released yet? this method was just introduced with 24b1b3b4c0e and it will be released in 3.16

Reply:
oops, sorry, indeed.

Reply:
The truth is I wanted to add another overload to preserve the API but was getting collisions on the python signatures. Maybe we should deprecate more variants of this method for QGIS 4 or refactor topologicalPoints as a member instead of a parameter.